TodoList/sources/TodoList.Services/UpdateNodeService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading.Tasks;
+using TodoList.Contracts.Models;
+using TodoList.Contracts.Repository;
+using TodoList.Contracts.Services;
+
+namespace TodoList.Services
+{
+    internal class UpdateNodeService : IUpdateNodeService
+    {
+        private readonly INodesRepository _repository;
+        private readonly ICurrentTimeService _timeService;
+
+        public UpdateNodeService(INodesRepository repository, ICurrentTimeService timeService)
+        {
+            _repository = repository;
+            _timeService = timeService;
+        }
+
+        public async Task<NodeModel> UpdateNodeAsync(NodeModel node)
+        {
+            var currentTime = _timeService.GetCurrentTime();
+
+            var chosenNode = await _repository.GetByIdAsync(node.Id);
```

Comment:
* Reminds me of Neo :rofl: 
* Try to be consistent in naming - I pointed out that you could use `existing` prefix in the `Controller`; whatever you decide to use, consider using it here as well
* Rename the `node` as well to express value's purpose or origin